### PR TITLE
Switch to using glide to manage package

### PR DESCRIPTION
**Pull request by taget**
This patch switches to using glide instead of godep, so that we don't need to using script to clone vendor packages.

### CODE REVIEW

File: docs/DeveloperQuickStart.md [hunk 1]
```diff
@@ -25,10 +25,14 @@ $ go get github.com/Masterminds/glide && glide install
 # generage configuration file
 $ go run cmd/gen_conf.go
 
+# install-deps.sh requires to using sudo access, also need go
+# so we need to amend go binary pach (e.g. `/usr/local/go/bin/go`) into
```
Comment (eliqiao) [inline docs/DeveloperQuickStart.md]: s/pach/path

File: docs/DeveloperQuickStart.md [hunk 2]
```diff
@@ -42,16 +46,27 @@ $ $GOPATH/bin/rmd
 
 Bash shell script `hacking.sh` checks coding style using `go fmt` and `golint`.
 
-Before you commit your changes, run `./hacking.sh` and address errors before you push your changes.
+Before you commit your changes, run `hacking.sh` in scripts directory,
+and address errors before you push your changes.
```
Comment (eliqiao) [inline docs/DeveloperQuickStart.md]: Add more description about using hacking, e.g. hacking.sh -f to do a full code checking

Comment (taget) [reply docs/DeveloperQuickStart.md]: Fixed.

File: docs/DeveloperQuickStart.md (hunk 1 above)
Comment (taget) [reply docs/DeveloperQuickStart.md]: Done.

**Pull request merged**